R/with.R

```diff
@@ -187,10 +187,11 @@ call_linter_factory <- function(linter_factory, linter_name, package) {
   linter <- tryCatch(
     linter_factory(),
     error = function(e) {
-      cli_abort(
```

Comment:
Revert this?

Reply:
Maybe we should return a "disabled" linter here too? Probably I'm overthinking it. Reverting for now.